infrastructure/laa-estimate-financial-eligibility-for-legal-aid-uat/permanent-redirects.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: check-client-qualifies-laa-estimate-eligibility-redirects
+  annotations:
+    external-dns.alpha.kubernetes.io/set-identifier: "check-client-qualifies-laa-estimate-eligibility-redirects-laa-estimate-financial-eligibility-for-legal-aid-uat-green"
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
+    nginx.ingress.kubernetes.io/rewrite-target: /
+    nginx.ingress.kubernetes.io/permanent-redirect: "https://check-your-client-qualifies-for-legal-aid.service.gov.uk"
+spec:
+  rules:
+    # Add rules for every host that you would like to redirect to production, like so:
+    - host: redirect-example-check-client-qualifies.cloud-platform.service.justice.gov.uk
+      http:
+        paths:
+          - path: /
+            pathType: ImplementationSpecific
+            backend:
+              service:
+                name: check-client-qualifies-laa-estimate-eligibility-redirects
```

Comment:
Is this a real service? Is it automatic?
Or is it just to have something and actually the annotation does the work and then the service of this ingress is never reached?